corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -245,6 +245,15 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
                           <option value="{{ value }}">{{ name }}</option>
                       {% endfor %}
                   </select>
+              </div>
+          </div>
+          <div class="form-group">
+              <label for="data-registry" class="{% css_label_class %} control-label">
+                  {% trans "Additional Data Registry Queries" %}
+                  <span class="hq-help-template" data-title="{% trans "Data Registry" %}"
+                        data-content="{% trans_html_attr "Load additional data into forms in this module by performing extra queries against the data registry." %}">
+              </label>
+              <div class="col-sm-9">
```

Comment:
This should either be `{% css_field_class %}` or the label above should be `col-sm-3`, so the two columns add up to 12 regardless of screen width.

Reply:
I was using `css_field_class` initially but it's quite squashed - I can revert to that though.

Reply:
eb3bab28fcc